recOrder/io/zarr_converter.py

```diff
@@ -19,31 +18,41 @@ class ZarrConverter:
     for tiled acquisitions)
     """
 
-    def __init__(self, input, output, data_type=None, replace_position_names=False, format_hcs=False):
+    def __init__(self, input_dir, output_dir, data_type=None, replace_position_names=False, format_hcs=False):
+        """
+
+        Parameters
+        ----------
+        input_dir: str
+            Input directory
+        output_dir: str
+            Output directory
+        data_type: str
+            input data type, optional
+        replace_position_names: bool
+        format_hcs: bool
+        """
 
         # Add Initial Checks
-        if len(glob.glob(os.path.join(input, '*.tif'))) == 0:
-            raise ValueError('Specific input contains no .tif files, please specify a valid input directory')
-        if not output.endswith('.zarr'):
+        # if len(glob.glob(os.path.join(input, '*.tif'))) == 0:
+        #     raise ValueError('Specific input contains no .tif files, please specify a valid input directory')
```

Comment:
Still worth keeping? I've seen this error as a user when I wasn't sure if I should give a file or a directory (your appending `_dir` to helps this problem), and I think I'd prefer this message over a cryptic IO failure if I accidentally supplied a folder without any tifs.

Reply:
I think this check should be implemented in the reader, rather than the converter, the reader should decide if the data format is valid. For example, pycromanager datasets are structures as `input_dir/Full Resolution/*.tif`. The waveorder `MicromanagerOmeTiffReader` implements this check here: https://github.com/mehta-lab/waveorder/blob/cbdba709594649a3eac50ea7c9adc8ec71c0d0cf/waveorder/io/multipagetiff.py#L23-L25